ignite/handlers/lr_finder.py

```diff
@@ -224,7 +224,10 @@ def plot(
 
         """
         try:
+            import matplotlib
             from matplotlib import pyplot as plt
+
+            matplotlib.use("tkagg")
```

Comment:
@OBITORASU do we really need that in the code ? I think either this should go to the tests or simply be removed...

Reply:
Trying some other approach then.